pkg/daemon/daemon.go

```diff
@@ -228,8 +223,19 @@ func (d *daemon) AddPeriodicUpdate() {
 				ibAnnotation = map[string]string{networkName: utils.ConfiguredInfiniBandPod}
 			}
 			ibAnnotationsData, _ := json.Marshal(ibAnnotation)
-
 			pod.Annotations[utils.InfiniBandAnnotation] = string(ibAnnotationsData)
+
+			network := podNetworkMap[pod.UID]
```

Comment:
how this will work if the pod as 2 networks of infiniband? it seem like you will only set the last one

Reply:
Ok it done for the same network which can contains different guid